src/api/handlers/guild.ts

```diff
@@ -797,3 +799,69 @@ export async function editGuildTemplate(
   ) as GuildTemplate;
   return structures.createTemplate(template);
 }
+
+function createMembershipStruct(
```

Comment:
Can't we use a resuable snakeToCamel function here? I'm sure well need it everywhere and might be easier to have 1 source instead of a function for each object

Reply:
@Skillz4Killz the `snakeToCamel()` function is not implemented yet, when it is, I'll change this.